README.md

```diff
@@ -1,8 +1,10 @@
 # Django Finite State Machine Log
 
-[![test suite](https://github.com/gizmag/django-fsm-log/actions/workflows/test_suite.yml/badge.svg)](https://github.com/gizmag/django-fsm-log/actions/workflows/test_suite.yml)
-[![Code Health](https://landscape.io/github/gizmag/django-fsm-log/master/landscape.png)](https://landscape.io/github/gizmag/django-fsm-log/master)
-[![codecov](https://codecov.io/gh/gizmag/django-fsm-log/branch/master/graph/badge.svg)](https://codecov.io/gh/gizmag/django-fsm-log)
+[![test suite](https://github.com/jazzband/django-fsm-log/actions/workflows/test_suite.yml/badge.svg)](https://github.com/jazzband/django-fsm-log/actions/workflows/test_suite.yml)
+[![codecov](https://codecov.io/gh/jazzband/django-fsm-log/branch/master/graph/badge.svg)](https://codecov.io/gh/jazzband/django-fsm-log)
+[![Jazzband](https://jazzband.co/static/img/badge.svg)](https://jazzband.co/)
+[![Documentation Status](https://readthedocs.org/projects/django-fsm-log/badge/?version=latest)](https://django-fsm-log.readthedocs.io/en/latest/?badge=latest)
+
 Automatic logging for the excellent [Django FSM](https://github.com/kmmbvnr/django-fsm)
```

Comment:
Although not strictly related, the original package is now located at https://github.com/viewflow/django-fsm (and we need to make a PR there too :-D)